packages/reporter/src/commands/command.tsx

```diff
@@ -358,6 +339,7 @@ class Command extends Component<Props> {
                 }
               </span>
               <span className='command-controls'>
+                {isSessionCommand && <Tag content={model.renderProps.status} type={model.renderProps.status === 'failed' ? 'failed' : 'passed'} />}
```

Comment:
Is `pending` a possible attribute here? It looks like that might be handled above but I am not sure.

Reply:
No pending is not, this session command is the only command (currently) that sets this status and it is meant to represent the session's workflow status - so `created`, `restored` and `recreated` are all "successful" statuses where as failed ties to the entire command failed....I agree this isn't overally clear/represented here, but I was trying to remove more one-off logic for sessions (though theres already so much 😅 ). Good feedback...Let me think on this.

Reply:
better? [`ecac805` (#22857)](https://github.com/cypress-io/cypress/pull/22857/commits/ecac80537b6bd1ba42758df2b927e0c2bf3a15f6#diff-fb0aaf9c972fca4aa414f95ac88ab9ad1619459edeab226100daaba5e55645d0R345)